app/dao/fact_notification_status_dao.py

```diff
@@ -242,6 +243,63 @@ def get_last_send_for_api_key(api_key_id):
     ).all()
 
 
+def get_api_key_ranked_by_notifications_created(n_days_back):
+    """
+    SELECT
+        a.api_key_id,
+        a.service_id,
+        a.last_send,
+        a.total_notifications,
+        api_keys.name,
+        api_keys.key_type,
+        services.name
+    FROM (
+        SELECT
+            api_key_id,
+            service_id,
+            max(created_at) as last_send,
+            count(*) as total_notifications
+        FROM notifications
+        WHERE created_at > start_date and api_key_id is not null
+        GROUP BY api_key_id, service_id
+        ORDER BY count(*) DESC
+        LIMIT 50
```

Comment:
This limit is arbitrary but I wanted to limit the size of the returned data. In the future it might make sense to do pagination if we want to view more than the top 50 keys or however many fit on the 1st page.